data/irisDataset.json

```diff
@@ -0,0 +1,152 @@
+[[5.1, 3.5, 1.4, 0.2, "setosa"],
```

Comment:
could you please remove this file and use `ml-dataset-iris` instead?

Reply:
It would be:

`npm install -D ml-dataset-iris`

```js
import { getNumbers } from 'ml-dataset-iris';
const rawData = getNumbers();
```

Reply:
the code has been modified accordingly